### PR TITLE
fix(init): Fix init for starters

### PR DESCRIPTION
**Pull request by xrutayisire**
## Context

- Completes DT-1386

## The Solution

- Fix init by detecting the starter used

## Impact / Dependencies

- Need PR from wroom to send starterId https://github.com/prismicio/wroom/pull/2470

## Checklist before requesting a review
- [x] I hereby declare my code ready for review.
- [x] If it is a critical feature, I have added tests.
- [x] The CI is successful.
- [x] If there could backward compatibility issues, it has been discussed and planned.

### CODE REVIEW

File: packages/init/src/SliceMachineInitProcess.ts [hunk 1]
```diff
@@ -270,23 +271,22 @@ export class SliceMachineInitProcess {
...
+								this.context.starterID = await detectStarterId(
+									this.manager.cwd,
+								);
+
+								if (this.context.starterID) {
+									task.title = `Detected starter ${chalk.cyan(
+										this.context.starterID,
+									)}`;
+								} else {
+									task.title = "No starter detected";
```
Comment (MarcMcIntosh) [inline packages/init/src/SliceMachineInitProcess.ts:286]: hmm, they'll no what a starter is so I guess this won't cause any panic

**Review by MarcMcIntosh — APPROVED** [review]
One comment about wording, but it's nothing major :)

File: packages/init/src/SliceMachineInitProcess.ts (hunk 1 above)
Comment (xrutayisire) [reply packages/init/src/SliceMachineInitProcess.ts:286]: You think I should change it to something less like a problem?

Maybe not even writing something?

Comment (xrutayisire) [reply packages/init/src/SliceMachineInitProcess.ts:286]: Discussed with Sam, it's ok like that 👍

Comment (lihbr) [reply packages/init/src/SliceMachineInitProcess.ts:286]: It appears for a split second anyway because the task drawer collapse right after

Comment (xrutayisire) [reply packages/init/src/SliceMachineInitProcess.ts:286]: Good one, true

**Pull request merged**